<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Gestión Académica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number> </property>
      
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Estudiantes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_estudiantes">
        <item>
         <widget class="QGroupBox" name="groupBox_registroEstudiante">
          <property name="title">
           <string>Registro de Estudiante</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_estudiante">
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lb_codigo_2">
             <property name="text">
              <string>Código (LLE999)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtCodigoEstudiante"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lb_codigo_3">
             <property name="text">
              <string>Nombre</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="txtNombreEstudiante"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_codigo_4">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txtEmailEstudiante"/>
           </item>
           <item row="1" column="2" rowspan="2">
            <widget class="QPushButton" name="btnAgregarEstudiante">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Agregar Estudiante</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_listaEstudiante">
          <property name="title">
           <string>Lista de Estudiantes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_busquedaEstudiante">
             <item>
              <widget class="QLineEdit" name="txtBuscarEstudiantes">
               <property name="placeholderText">
                <string>Buscar por Código, Nombre o Email...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBuscarEstudiantes">
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tblEstudiantes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Cursos</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_cursos">
        <item>
         <widget class="QGroupBox" name="groupBox_registroCurso">
          <property name="title">
           <string>Registro de Curso</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_curso">
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lb_codigo">
             <property name="text">
              <string>Código (LLE999)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtCodigoCurso"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lb_nombreCurso">
             <property name="text">
              <string>Nombre del Curso</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="txtNombreCurso"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_fechaCurso">
             <property name="text">
              <string>Fecha de Creación</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="dtFechaCursos">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yyyy</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" rowspan="2">
            <widget class="QPushButton" name="btnAgregarCurso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Agregar Curso</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_listaCurso">
          <property name="title">
           <string>Lista de Cursos</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_busquedaCurso">
             <item>
              <widget class="QLineEdit" name="txtBuscarCursos">
               <property name="placeholderText">
                <string>Buscar por Código o Nombre...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBuscarCursos">
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tblCursos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Notas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_notas">
        <item>
         <widget class="QGroupBox" name="groupBox_registroNota">
          <property name="title">
           <string>Registro de Nota</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_notas">
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lb_codNota">
             <property name="text">
              <string>Estudiante (Código)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtCodigoEstudianteNota"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lb_codCurso">
             <property name="text">
              <string>Curso</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cbCursos"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_nota">
             <property name="text">
              <string>Nota (0.0 - 20.0)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txtNota"/>
           </item>
           <item row="1" column="2" rowspan="2">
            <widget class="QPushButton" name="btnAgregarNota">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Agregar Nota</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_listaNota">
          <property name="title">
           <string>Lista de Notas</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_busquedaNota">
             <item>
              <widget class="QLineEdit" name="txtBuscarNotas">
               <property name="placeholderText">
                <string>Buscar por Código de Estudiante...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBuscarNotas">
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tblNotas">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Asistencia</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_asistencia">
        <item>
         <widget class="QGroupBox" name="groupBox_registroAsistencia">
          <property name="title">
           <string>Registro de Asistencia</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_asistencia">
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lb_codAsis">
             <property name="text">
              <string>Código Estudiante</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtCodigoEstudianteAsis"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lb_codCursoAsis">
             <property name="text">
              <string>Curso</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cbCursosAsistencia"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lb_fecha">
             <property name="text">
              <string>Fecha:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_fecha">
             <item>
              <widget class="QDateEdit" name="dtFechaAsis">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="displayFormat">
                <string>dd/MM/yyyy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkPresente">
               <property name="text">
                <string>Presente</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QPushButton" name="btnRegistrarAsistencia">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Registrar Asistencia</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_listaAsistencia">
          <property name="title">
           <string>Lista de Asistencias</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_busquedaAsistencia">
             <item>
              <widget class="QLineEdit" name="txtBuscarAsistencias">
               <property name="placeholderText">
                <string>Buscar por Código de Estudiante o Curso...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBuscarAsistencias">
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tblAsistencias">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>